UI/window-basic-status-bar.cpp

```diff
@@ -233,11 +225,30 @@ void OBSBasicStatusBar::UpdateCPUUsage()
 
 	QString text;
 	text += QString("CPU: ") +
-		QString::number(main->GetCPUUsage(), 'f', 1) + QString("%, ") +
-		QString::number(obs_get_active_fps(), 'f', 2) + QString(" fps");
+		QString::number(main->GetCPUUsage(), 'f', 1) + QString("%");
+
+	statusWidget->ui->cpuUsage->setText(text);
+
+	UpdateCurrentFPS();
+}
 
-	cpuUsage->setText(text);
-	cpuUsage->setMinimumWidth(cpuUsage->width());
+void OBSBasicStatusBar::UpdateCurrentFPS()
+{
+	OBSBasic *main = qobject_cast<OBSBasic *>(parent());
+	if (!main)
+		return;
+
+	struct obs_video_info ovi;
+	obs_get_video_info(&ovi);
+	float targetFPS = (float)ovi.fps_num / (float)ovi.fps_den;
+
+	QString text;
+	text += QString::number(obs_get_active_fps(), 'f', 2);
+	text += QString(" / ");
+	text += QString::number(targetFPS, 'f', 2);
+	text += QString(" FPS");
+
```

Comment:
What is the point of showing the selected output FPS in the status bar? This is a static value once set, so doesn't really belong in a status bar IMO. The current FPS will also match the output FPS in the majority of cases so this looks confusing to the user ("why do I have two FPS numbers?").

Also this seems to be missing on the main screenshot of the PR - UI changes should always have accurate screenshots.

Reply:
The screenshot has now been updated.

Reply:
We went with `current / max FPS` after Fenrir suggested switching to `render time / max render time ms` upon clicking. I think it's good to maintain the direct link between both. This behaviour is not implemented in this PR, but eventually it will.